records_mover/records/schema/__init__.py

```diff
@@ -1,2 +1,18 @@
-from .errors import DeserializationError, UnsupportedSchemaError  # noqa
-from .schema import RecordsSchema  # noqa
+__all__ = [
+    'RecordsSchemaKnownRepresentation',
+    'RecordsSchema',
+    'DeserializationError',
+    'UnsupportedSchemaError',
+    'RecordsSchemaField',
+    'RecordsSchemaFieldConstraints',
+    'RecordsSchemaFieldStatistics',
+    'RecordsSchemaFieldRepresentation',
+]
```

Comment:
I started down the path of bringing this stuff into the public API but ultimately punted.  At least we can have simpler imports of it in the meantime.